recipes/parmoo/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "parmoo" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/parmoo-{{ version }}.tar.gz
+  sha256: 5a2606665783b63915913f96393876c44c85c93259153842c3482d68a9ea32b1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - tox
```

Comment:
Is tox actually necessary to build the package?
```suggestion
```

Reply:
no, tox is not required.  I missed that.  I think it was mistakenly grabbed by grayskull

Reply:
removed it in the latest push